packages/website/ts/components/documentation/tutorial_button.tsx

```diff
@@ -0,0 +1,79 @@
+import { colors, Link } from '@0xproject/react-shared';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Text } from 'ts/components/ui/text';
+import { Deco, Key, TutorialInfo } from 'ts/types';
+import { Translate } from 'ts/utils/translate';
+
+export interface TutorialButtonProps {
+    translate: Translate;
+    tutorialInfo: TutorialInfo;
+}
+
+export interface TutorialButtonState {
+    isHovering: boolean;
+}
+
+export class TutorialButton extends React.Component<TutorialButtonProps, TutorialButtonState> {
+    constructor(props: TutorialButtonProps) {
+        super(props);
+        this.state = {
+            isHovering: false,
+        };
+    }
+    public render(): React.ReactNode {
+        return (
+            <Link
+                to={this.props.tutorialInfo.link.to}
+                shouldOpenInNewTab={this.props.tutorialInfo.link.shouldOpenInNewTab}
+                onMouseEnter={this._onHover.bind(this)}
+                onMouseOver={this._onHover.bind(this)}
+                onMouseLeave={this._onHoverOff.bind(this)}
+            >
+                <div
+                    className="flex relative"
+                    style={{
+                        borderRadius: 4,
+                        border: `1px solid ${this.state.isHovering ? colors.lightLinkBlue : colors.grey325}`,
+                        padding: 20,
+                        marginBottom: 15,
+                        backgroundColor: this.state.isHovering ? colors.lightestBlue : colors.white,
+                    }}
```

Comment:
You could have this be a stateless component and remove all the `onMouseLeave` `onMouseEnter` `onMouseOver` logic if it's only being used to modify `isHovering` and `isHovering` is only being used to change styles. With `styled-components` you can use pseudo selectors. See the `<Button/>` component for example.